docs/user/canvas.asciidoc

```diff
@@ -18,16 +18,170 @@ With Canvas, you can:
 * Focus the data you want to display with filters.
 
 [role="screenshot"]
-image::images/canvas-gs-example.png[]
+image::images/canvas-gs-example.png[Getting started example]
 
 For a quick overview of Canvas, watch link:https://www.youtube.com/watch?v=ZqvF_5-1xjQ[Stand out with Canvas].
-//When https://github.com/elastic/Video/issues/358 is resolved, update this link.
+
+[float]
+[[create-workpads]]
+== Create workpads
+
+A _workpad_ provides you with a workspace where you can build presentations of your live data.
```

Comment:
Is there a way to describe a workpad without using "workspace"?  They sound so similar.